src/controllers/comment.js

```diff
@@ -1,4 +1,4 @@
-import model from "../database/models";
+import model from '../database/models';
 
 export const CreateComment = async (req, res) => {
```

Comment:
Anyone can add a comment on any request?

Reply:
no, you need to specify requestId